### PR TITLE
Faves images

### PR DESCRIPTION
**Pull request by CrowRising**
____ Wrote Tests ____ Implemented ____ Reviewed

Neccesary checkmarks:

  - [x] All Tests are Passing

  - [x] The code will run locally

Type of change

  - [x] New feature
  - [ ] Bug Fix

Implements/Fixes:

    

Check the correct boxes

  - [x] This broke nothing
  - [ ] This broke some stuff
  - [ ] This broke everything

Testing Changes

   - [ ] No Tests have been changed
   - [x] Some Tests have been changed
   - [ ] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

   - [x] My code has no unused/commented out code
   - [x] I have reviewed my code
   - [ ] I have commented my code, particularly in hard-to-understand areas
   - [x] I have fully tested my code

broken link

### CODE REVIEW

**Comment by thawley2** [comment]
Trying to set up circle ci

**Comment by CrowRising** [comment]
Okay, since we are ditching circle CI for back end here is that pull request from this morning completing the user's favorite playground request

File: app/controllers/api/v0/user_playgrounds_controller.rb [hunk 1]
```diff
@@ -1,17 +1,16 @@
 class Api::V0::UserPlaygroundsController < ApplicationController
     def index
-        @user_playgrounds = UserPlayground.all
-        render json: @user_playgrounds
+        @user_playgrounds = UserPlayground.where(user_id: params[:user_id])
```
Comment (thawley2) [inline app/controllers/api/v0/user_playgrounds_controller.rb:3]: Nicely done with the serializer and AR

File: spec/requests/api/v0/user_playground_request_spec.rb [hunk 2]
```diff
@@ -5,21 +5,22 @@
...
-        expect(user_playground).to have_key('id')
-        expect(user_playground['id']).to be_a(Integer)
-        expect(user_playground).to have_key('user_id')
-        expect(user_playground['user_id']).to be_a(Integer)
-        expect(user_playground).to have_key('playground_id')
-        expect(user_playground['playground_id']).to be_a(Integer)
-        expect(user_playground).to have_key('playground_name')
-        expect(user_playground['playground_name']).to be_a(String)
+ 
+      user_playgrounds = JSON.parse(response.body, symbolize_names: true)
```
Comment (thawley2) [inline spec/requests/api/v0/user_playground_request_spec.rb:13]: Nice refactor of the tests!

**Review by thawley2 — APPROVED** [review]
Looks great, just one more endpoint to hit!!

**Pull request merged**